llvm/test/CodeGen/NVPTX/surf-tex.py

```diff
@@ -353,8 +364,9 @@ def gen_sust_tests(target, global_surf):
     ret void
   }
   ; CHECK-LABEL: .entry ${test_name}_global
-  ; CHECK: ${instruction} [${global_surf}, ${reg_access}], ${reg_value}
-  ;
+  ; CHECK-CUDA: mov.u64 [[REG${reg_id}:%.*]], ${global_surf}
+  ; CHECK-CUDA: ${instruction} [[[REG${reg_id}]], ${reg_access}], ${reg_value}
```

Comment:
I wonder why we end up with an extra `mov` for CUDA, but not openCL. 
The issue is cosmetic, and is not a blocker for this patch.